src/Selenium2Library/keywords/browsermanagement.py

```diff
@@ -556,8 +581,7 @@ def _make_browser(self, browser_name, desired_capabilities=None,
 
         return browser
 
-
-    def _make_ff(self , remote , desired_capabilites , profile_dir):
+    def _make_ff(self, remote, desired_capabilites, profile_dir):
```

Comment:
+1 for removing unnecessary and PEP-8 incompatible whitespace. Separate commit in master would have been better, though. I do this all the time too and I wouldn't revert these changes.